src/azure-cli/azure/cli/command_modules/hdinsight/tests/latest/test_hdinsight_commands.py

```diff
@@ -605,7 +605,7 @@ def _kafka_arguments():
     def _rest_proxy_arguments():
         return '--kafka-management-node-size {} --kafka-client-group-id {} --kafka-client-group-name {} -v 4.0 ' \
                '--component-version {} --location {}' \
-            .format('Standard_D4_v2', '7bef90fa-0aa3-4bb4-b4d2-2ae7c14cfe41', 'KafakaRestProperties', 'kafka=2.1',
+            .format('Standard_D4_v2', '7bef90fa-0aa3-4bb4-b4d2-2ae7c14cfe41', 'KafkaRestProperties', 'kafka=2.1',
```

Comment:
@yonzhan, this is your turf. 🤣

Reply:
Good catch! It should be Kafka, of course.